src/internal/orders/sales-order-identifier.ts

```diff
@@ -4,11 +4,13 @@ import { hideAndFreeze, Identifiers, Joi, _internal } from "../common";
 
 export abstract class SalesOrderIdentifierBase implements ISalesOrderIdentifier {
   public readonly id: string;
+  public readonly orderNumber?: string;
```

Comment:
@justin-shipstation Make this required, instantiate in the constructor as 

```javascript
this.orderNumber = pojo.orderNumber || ""
```

Reply:
Updated